### PR TITLE
DEIMOS blue dev

### PR DESCRIPTION
**Pull request by profxj**
Couples with this PR:  https://github.com/pypeit/PypeIt/pull/1059

Adds wavelengths and new tests for `develop`

### CODE REVIEW

File: dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*.dat files
+IRAF outputs from Carlos (WMKO)
```
Comment (jhennawi) [inline dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README:2]: Should we convert the template to vacuum?

File: test_scripts/test_setups.py [hunk 2]
```diff
@@ -105,8 +105,11 @@ class TestPhase(Enum):
 
 develop_setups = {'shane_kast_blue': ['452_3306_d57', '600_4310_d55', '830_3460_d46'],
                   'shane_kast_red': ['600_7500_d55_ret', '600_7500_d57', '600_5000_d46', '1200_5000_d57'],
+                  'gemini_gnirs': ['32_SB_SXD', '10_LB_SXD'],
+                  'gemini_gmos': ['GS_HAM_R400_700', 'GS_HAM_R400_860', 'GN_HAM_R400_885'],
+                  'gemini_flamingos2': ['HK_HK', 'JH_JH'],
                   'keck_deimos': ['600ZD_M_6500', '600ZD_tilted', '1200G_M_7750', '830G_LVM_8400', '830G_M_8100_26',
-                                  '830G_M_8500', '830G_L_8100'],
+                                  '830G_M_8500', '830G_L_8100', '1200B_Percy', '1200G_Ghassem'],
```
Comment (jhennawi) [inline test_scripts/test_setups.py:112]: Can we change the naming convention here to abide by the convention we already established?

File: test_scripts/test_setups.py [hunk 3]
```diff
@@ -119,22 +122,19 @@ class TestPhase(Enum):
                                     'multi_400_8500_d560',
                                     'long_400_8500_longread'],  # Longslit read-out mode
                   'keck_lris_red_orig': ['long_300_5000'],
-                  'vlt_xshooter': ['VIS_1x1', 'VIS_2x1', 'VIS_2x2', 'VIS_manual', 'NIR'],
```
Comment (jhennawi) [inline test_scripts/test_setups.py:122]: I am not following why you are removing these gemini tests?

File: test_scripts/test_setups.py [hunk 4]
```diff
@@ -119,22 +122,19 @@ class TestPhase(Enum):
...
                   'magellan_fire': ['FIRE_Echelle', 'FIRE_Long'],
                   'mdm_osmos': ['MDM4K'],
+                  'mmt_binospec': ['Longslit_G600', 'Multislit_G270'],
+                  'mmt_mmirs': ['HK_zJ', 'J_zJ', 'K_K'],
                   'not_alfosc': ['grism4', 'grism19'],
                   'p200_dbsp_blue': ['600_4000_d55'],
                   'p200_dbsp_red': ['316_7500_d55'],
                   'p200_tspec': ['TSPEC'],
                   'vlt_fors2': ['300I'],
-                  'lbt_luci': ['LUCI-I', 'LUCI-II'],
```
Comment (jhennawi) [inline test_scripts/test_setups.py:134]: Similar comment here, why are these tests being removed from develop. I disagree with that.

**Review by jhennawi — CHANGES_REQUESTED** [review]
Hi @profxj looks great. My main comment is why you have changed the develop test suite?

File: dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README (hunk 1 above)
Comment (profxj) [reply dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README:2]: they were

File: test_scripts/test_setups.py (hunk 2 above)
Comment (profxj) [reply test_scripts/test_setups.py:112]: like what?

File: test_scripts/test_setups.py (hunk 4 above)
Comment (profxj) [reply test_scripts/test_setups.py:134]: look again

File: test_scripts/test_setups.py (hunk 2 above)
Comment (jhennawi) [reply test_scripts/test_setups.py:112]: Grating name, decker ('M' for multisit), central wavelength.

File: dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README (hunk 1 above)
Comment (jhennawi) [reply dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README:2]: maybe make that clear in the README. .

**Review by jhennawi — APPROVED** [review]

File: test_scripts/test_setups.py (hunk 2 above)
Comment (profxj) [reply test_scripts/test_setups.py:112]: done

**Comment by profxj** [comment]
(base) profx> ./pypeit_test -t 5 -i keck_deimos develop
Running tests on the following instruments:
    keck_deimos

Reducing data from keck_deimos for the following setups:
    600ZD_M_6500
    600ZD_tilted
    1200G_M_7750
    830G_LVM_8400
    830G_M_8100_26
    830G_M_8500
    830G_L_8100
    1200B_M_5200
    1200G_M_5500

Running tests in 5 parallel processes
 0 passed/ 0 failed/ 0 skipped STARTED keck_deimos/830G_M_8500 pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED keck_deimos/600ZD_M_6500 pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED keck_deimos/1200G_M_7750 pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED keck_deimos/600ZD_tilted pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED keck_deimos/830G_L_8100 pypeit (without masters)
 1 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/830G_L_8100 pypeit (without masters)
 1 passed/ 0 failed/ 0 skipped STARTED keck_deimos/830G_LVM_8400 pypeit (without masters)
 2 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/600ZD_tilted pypeit (without masters)
 2 passed/ 0 failed/ 0 skipped STARTED keck_deimos/830G_M_8100_26 pypeit (without masters)
 3 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/1200G_M_7750 pypeit (without masters)
 3 passed/ 0 failed/ 0 skipped STARTED keck_deimos/1200B_M_5200 pypeit (without masters)
 4 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/830G_LVM_8400 pypeit (without masters)
 4 passed/ 0 failed/ 0 skipped STARTED keck_deimos/1200G_M_5500 pypeit (without masters)
 5 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/830G_M_8100_26 pypeit (without masters)
 6 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/600ZD_M_6500 pypeit (without masters)
 7 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/830G_M_8500 pypeit (without masters)
 8 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/1200G_M_5500 pypeit (without masters)
 9 passed/ 0 failed/ 0 skipped PASSED  keck_deimos/1200B_M_5200 pypeit (without masters)

--- PYPEIT DEVELOPMENT SUITE PASSED 9/9 TESTS (Masters ignored) ---
Testing Started at 2020-11-13T06:01:18.708667
Testing Completed at 2020-11-13T07:34:15.198018
Total Time: 1:32:56.489351

**Comment by profxj** [comment]
merging

**Pull request merged**